server/plugins/renderer/renderer.module.js

```diff
@@ -198,7 +201,53 @@ internals.parseResponse = async (bcAppData, request, response, responseArgs) =>
         cache.put(dataRequestSignature, { response2 }, internals.cacheTTL);
     }
 
-    return internals.getPencilResponse(response2.data, request, response2, configuration);
+    const templateFile = response2.data.template_file;
+    const entityId = response2.data.entity_id;
+
+    const pageType = getPageType(templateFile);
+    let regionResponse = [];
+
+    if (pageType !== null) {
```

Comment:
`if (pageType) { ... }`?

What are the possibilities here?

Reply:
I am setting the value to `null` specifically in `getPageType`. But I can see using just `if (pageType) { ... }`.